cli/webfinger.ts

```diff
@@ -0,0 +1,84 @@
+import { Command } from "@cliffy/command";
+import { lookupWebFinger } from "@fedify/fedify/webfinger";
+import ora from "ora";
+import { printJson } from "./utils.ts";
+
+export const command = new Command()
+  .arguments("<handle:string>")
+  .description(
+    "Lookup a WebFinger resource by handle. The argument can be multiple.",
```

Comment:
It's trivial, still …

```suggestion
    "Look up a WebFinger resource by handle. The arguments can be multiple.",
```

Reply:
Changed at bbbd59aedf0b22859a9214aa4104c2e3a683fe5f!